hw_twitterapi/twitterapiapp/views.py

```diff
@@ -193,14 +195,50 @@ def recent_favorites_api(request, screen_name):
 
 
 def proc_recent_favorites(request, screen_name):
-
     api = twitter_auth()
     tweet_count, retweet_count = 0, 0
 
     # print out each favorited tweet
     for page in tweepy.Cursor(api.favorites, id=screen_name, wait_on_rate_limit=True, count=200).pages(200):
-
         for status in page:
             tweet_count, retweet_count = tweet_count+1, retweet_count + status.retweet_count
 
     return {'tweet_count': tweet_count, 'retweet_count': retweet_count, 'retweet_average': retweet_count/tweet_count}
+
+def find_followers(request, follower):
+    context = {}
+    if proc_find_followers_api(follower):
+        context['Followers'] = proc_find_followers_api(follower)
```

Comment:
repeating `proc_find_followers_api` function may drain api limits. it's better to hold it in a variable.